packages/components/src/tip/index.js

```diff
@@ -4,21 +4,42 @@
 import { SVG, Path } from '@wordpress/primitives';
 
 /**
- * @typedef Props
- * @property {import('react').ReactNode} children Children to render in the tip.
+ * Internal dependencies
  */
+import useShadowStyles, { css } from '../utils/hooks/use-shadow-styles';
+import { ALERT, COLORS } from '../utils/colors-values';
+import { space } from '../utils/space';
+
+const styles = css`
+	:host {
+		display: flex;
+		color: ${ COLORS.gray[ 700 ] };
+	}
+
+	::slotted( svg ) {
```

Comment:
With the current eslint config, VSCode warns about both `::slotted( svg )` and `::slotted(svg)`. For the long run we would have to update the lining config as well.